### DISCUSSION TITLE
Ask smoothness quest immediately after surface quest; and surface immediately after sidewalk type

### DISCUSSION THREAD

**Discussion by tordans** [streetcomplete/StreetComplete#4593] [Ideas]
I have a setup where I have the sidewalk overlay active + all surface & smoothness quests. This is great to evaluate an area while taking a walk at tagging a similar category of road data.

I see an opportunity to improve UX for power users by reducing clicks but also for novice users by guiding them through the UI. For novice users, I found that understanding the interaction points of overlay + quests is really hard.

Ideas:

1. When surface and smoothness quests are both on, ask for smoothness right after surface (without the need to tab the quest pin "a second time")

2. When the sidewalk overlay is on and also the sidewalk surface (and possibly smoothness in the future) quest, ask for those data right after the sidewalk overlay was confirmed for a road segment.

Both improvements will reduce the number of tabs and also reduce the need to understand details about the UI and how the app works.

**Comment by mnalis** [comment]
Related: [StreetComplete Expert Edition](https://github.com/Helium314/StreetComplete/releases) fork has (1) implemented; but one needs to enable it in `Settings` / `EE Settings` / `UI` / `Show next quest immediately` first.
